### ISSUE TITLE
ConnectionError - Eli5 dataset download

### ISSUE THREAD

**Issue by Jupiter79** [huggingface/nlp#290]
Hi, I have a problem with downloading Eli5 dataset. When typing `nlp.load_dataset('eli5')`, I get ConnectionError: Couldn't reach https://storage.googleapis.com/huggingface-nlp/cache/datasets/eli5/LFQA_reddit/1.0.0/explain_like_im_five-train_eli5.arrow

I would appreciate if you could help me with this issue.

**Comment by lhoestq** [comment]
It should ne fixed now, thanks for reporting this one :)
It was an issue on our google storage.

Let me now if you're still facing this issue.

**Comment by Jupiter79** [comment]
It works now, thanks for prompt help!
